### PR TITLE
Eliminate :contains

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

File: src/ostree.jsx [hunk 1]
```diff
@@ -629,7 +632,7 @@ const OStreeSource = ({ ostreeState, refreshRemotes, onChangeBranch, onChangeRem
     const Dialogs = useDialogs();
 
     const actions = [
-        <DropdownItem key="rebase"
+        <DropdownItem key="rebase" data-action="rebase"
```
Comment (jelly) [inline src/ostree.jsx:635]: Would be nice if the key leaked through to the DOM right ;-)

**Review by jelly — APPROVED** [review]
Assuming this goes green, nice!

I was looking at this briefly, and finding something for the dropdownitem seemed annoying, but `data-action` actually seems nice!

The tab solution is alright, not superb, but it doesn't seem like our table supports setting any id / description on the tab. Maybe something to consider but then again asserting that the tab position is 1,2,3 is already great! (And also a test of sorts)

**Review by jelly — COMMENTED** [review]
Hmm what we just merged is flaky? That's strange!

**Comment by martinpitt** [comment]
Ah no, much simpler -- `become_superuser()` requires `:contains()`, so it's dropping sizzle that broke it. I'll fix that tomorrow.

File: src/ostree.jsx (hunk 1 above)
Comment (martinpitt) [reply src/ostree.jsx:635]: heh yea, but it's only a React detail..

**Comment by martinpitt** [comment]
I dropped the "remove sizzle" commit for now. I'll make test/common free of :contains tomorrow, and can then drop sizzle for good.

**Comment by jelly** [comment]
> I dropped the "remove sizzle" commit for now. I'll make test/common free of :contains tomorrow, and can then drop sizzle for good.

Aaaah, that explains a lot :) Was there a clue in the logs? :)

**Review by jelly — APPROVED** [review]
Hurray!

**Comment by martinpitt** [comment]
The clue was to stare at

> self.wait_in_text("div[role=dialog]:contains('Switch to administrative access')", f"Password for {user}:")

for a few seconds and realize the `:contains` :grin:

**Pull request merged**